config/settings/development.yml

```diff
@@ -1 +1,2 @@
 homepage_url: '//sumofus.org'
+host: "http://localhost"
```

Comment:
This is used already in production (the host is used to build a page's URI for sending to AK)